arbor/builtin_mechanisms.cpp

```diff
@@ -9,21 +9,9 @@
 
 namespace arb {
 
-template <typename B>
-concrete_mech_ptr<B> make_builtin_stimulus();
-
 mechanism_catalogue build_builtin_mechanisms() {
```

Comment:
Are we sure we cannot delete this completely?

Reply:
Well, we could! I just didn't want to be hasty. Different PR perhaps?